core/templates/dev/head/app.js

```diff
@@ -195,6 +195,15 @@ oppia.factory('oppiaDatetimeFormatter', [function() {
         return date.toLocaleTimeString().replace(/:\d\d /, ' ');
       }
       return date.toLocaleDateString();
+    },
+    // Returns just the date.
+    getLocaleAbbreviatedDateString: function(millisSinceEpoch) {
+      var date = new Date(millisSinceEpoch);
+      if (date.toLocaleDateString() == new Date().toLocaleDateString()) {
+        // If the date is the same, return current date.
+        return new Date().toLocaleDateString();
+      }
+      return date.toLocaleDateString;
```

Comment:
Ignore my previous comment. I see your suggestion now!

"I suggest adding another getLocaleDateString() method in oppiaDatetimeFormatter that's a simple wrapper around date.toLocaleDateString(), and adding a couple of tests for that in appSpec.js as well following the example that's already there."